packages/next/client/components/react-dev-overlay/hot-reloader-client.tsx

```diff
@@ -144,7 +144,7 @@ function tryApplyUpdates(
       return
     }
 
-    const hasUpdates = Boolean(updatedModules?.length)
```

Comment:
Removed these optional chaining checks where there is already a null check on `updatedModules`.